FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -588,8 +597,11 @@ TickType_t uxWriteTimeOut_ticks = ipconfigDNS_SEND_BLOCK_TIME_TICKS;
 
 	if( xDNSSocket != NULL )
 	{
-		FreeRTOS_setsockopt( xDNSSocket, 0, FREERTOS_SO_SNDTIMEO, ( void * ) &uxWriteTimeOut_ticks, sizeof( TickType_t ) );
-		FreeRTOS_setsockopt( xDNSSocket, 0, FREERTOS_SO_RCVTIMEO, ( void * ) &uxReadTimeOut_ticks,  sizeof( TickType_t ) );
+		/* Ideally we should check for the return value. But since we are passing
+		 * correect parameters, and xDNSSocket is != NULL, the return value is 
+		 * going to be '0' i.e. success. Thus, return value is discarded */
```

Comment:
Seems like we should check the return value.

Reply:
I agree that we should. But I also think that it will be an overkill since this is an internal call and we are passing correct parameters with a valid socket (checked here `if( xDNSSocket != NULL )`). I do not thing that there can be an error in the function. The function can be found [here](https://github.com/FreeRTOS/FreeRTOS/blob/master/FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_Sockets.c#L1282). It is a big switch case which returns `0` (success) in all cases except when an invalid parameter is sent to the function.
Do you agree?

Reply:
done